src/Command/DatabaseCommand.php

```diff
@@ -262,4 +260,12 @@ private function excludeParts(string $database): string
         }
         return '';
     }
+
+    private function getCombinationPipe(string $targetDatabase): string
+    {
+        $escapedTargetDatabase = '`' . str_replace('`', '``', $targetDatabase) . '`';
+        $intermediateSql = sprintf('CREATE DATABASE IF NOT EXISTS %s;USE %s;', $escapedTargetDatabase, $escapedTargetDatabase);
+        $combinationPipe = ' | (echo ' . escapeshellarg($intermediateSql) . ' && cat) | ';
```

Comment:
`return ' | (echo ' . escapeshellarg($intermediateSql) . ' && cat) | ';`

Reply:
I would like to merge this PR before we change anything else.